README.md

```diff
@@ -6,33 +6,90 @@
+
+[kubevirt migration toolkit](https://github.com/kubev2v/forklift) (Forklift) is a suite of migration tools that facilitate the import of virtualization workloads to [OpenShift Virtualization](https://cloud.redhat.com/learn/topics/virtualization/).
 
 ## Prerequisites
 
-* [__Forklift Operator__](https://github.com/kubev2v/forklift-operator/)
+* [__Forklift Operator 2.4.0+__](https://github.com/kubev2v/forklift/)
 * [__OpenShift Console 4.12+__](https://www.openshift.com/)
 
-## Development
+## Quick start
+
+forklift-console-plugin is automatically installed when installing the [Forklift operator](https://github.com/kubev2v/forklift/). A user can also install the plugin manually using [Helm](https://helm.sh/) on a cluster with forklift operator already installed.
+
+When forklift-console-plugin is installed, [Openshift web console](https://github.com/openshift/console) will add new menu item under the `virtualization` secition for virtualization workloads import.
+
+The folowing script will use [Helm](https://helm.sh/) to install the plugin on a cluster running forklift controller:
+
+``` bash
+# add the forklift helm repo
+helm repo add forklift https://kubev2v.github.io/forklift-console-plugin
+
+# install the chart using the repository package
+helm install forklift-console-plugin forklift/forklift-console-plugin \
+  --set forklift.namespace=<the forklift operator namespace>
+```
+
+For more information about forklift-console-plugin deployment, see the [forklift-console-plugin docs](https://github.com/kubev2v/forklift-console-plugin/tree/main/docs).
+
+## Requirements
+
+forklift-console-plugin development require web development tools, and a kubernetes or Openshift cluster. You can use any available [openshift](https://www.openshift.com/) cluster, or deploy your own small development cluster on your local PC, using [OpenShift Local](https://developers.redhat.com/products/openshift-local) or [KinD](https://sigs.k8s.io/kind).
 
-### Start the UI:
+| requirements        |     |
+|---|----|
+| nodejs | JavaScript runtime environment |
+| yarn | package manager for nodejs |
+| kubernetes | An [Openshift]((https://www.openshift.com/)) or kubernetes cluster for development |
+
+| optional        |     |
+|---|----|
+| kubectl | The Kubernetes command-line tool |
+| forklift | [forklift](https://github.com/kubev2v/forklift/) controller running on the development cluster |
+| kubevirt | [kubevirt](https://kubevirt.io/) controller running on the development cluster |
+
+For more information about forklift-console-plugin deployment enviorment, see the [forklift-console-plugin docs](https://github.com/kubev2v/forklift-console-plugin/tree/main/docs).
+
+### Spin your own small kubernetes
+
+This example script is using [KinD](https://sigs.k8s.io/kind) to deploy a minimal kubernetes cluster for local development:
+
+```bash
+# Requires kind installed on you machine,
+bash scripts/deploy-kind.sh
```

Comment:
> install kind is via go install as suggested in docs/cli-tools.md#kind

Added a comment about installing kind, also renamed to `deploy-cluster` to make sure users don't think this script will install kind

p.s.
I updated the documentation in `cli-tools.md`,
no need to install `go`, install instruction now use `curl`

> Error: can't find 'kind' command, exit

install now to `/usr/local/bin` so it's available to all users

Reply:
> due to rootlessport blocking the port (clean-kind.sh didn't help)

fixed the `clean-cluster.sh` script, to not exist if kind is missing, now, it will remove the registry, even if kind is not installed